src/pages/LogoOrganizations.tsx

```diff
@@ -184,7 +184,7 @@ class LogoOrganizations extends Component<Props, LogoState> {
                       logoIsValid
                     }
                     isValid={logoIsValid}
-                    logoPath={`${LOGO_PATH}/organization/${logoId}`}
+                    logoPath={`${ORGANIZATIONS_LOGO_PATH}/organization/${logoId}`}
```

Comment:
Since `/organizations` is already present in `ORGANIZATIONS_LOGO_PATH` we should remove it.

Reply:
oops, you're right